spec/classes/dnat4_spec.rb

```diff
@@ -56,71 +56,71 @@
             owner:          'root',
             group:          'root',
             mode:           '0640',
-            ensure_newline: true,
```

Comment:
FYI: we had some discussion about trailing comma's. I think most people agreed that we should enable them in Rubocop as well. We use them in Puppet so it's inconsistent not to do the same in Ruby. I would prefer that over changing everything in this module.

Reply:
Agreed.